ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -66,21 +66,29 @@ def getFTS2ServersForSites( self, siteList = None ):
 
 
 def getFTS3Servers():
-  """ get FTSServers for sites
-
-  :param list siteList: list of sites
+  """ get FTS3 Server URL
   """
 
   csPath = cfgPath( gBaseResourcesSection, "FTSEndpoints/FTS3" )
-  # We do it in two times to keep the order
-  ftsServerNames = gConfig.getOptions( csPath ).get( 'Value', [] )
-
+  # retrieving list of params
+  params = gConfig.getOptions( csPath ).get( 'Value', [] )
```

Comment:
Just curiosity, but why do you change 'ftsServerNames', which is a meaningful name in 'params', which does not really help in knowing what it is ?